### ISSUE TITLE
"current_buffer_fuzzy_find" error on buffers where highlight by treesitter is enabled.

### ISSUE THREAD

**Issue by sabah1994** [nvim-telescope/telescope.nvim#2279]
### Description

When trying to invoke "current_buffer_fuzzy_find" I get following error:
Error executing Lua callback: ...plugged/telescope.nvim/lua/telescope/builtin/__files.lua:413: attempt to call method '_get_hl_from_capture' (a nil value)
stack traceback:
        ...plugged/telescope.nvim/lua/telescope/builtin/__files.lua:413: in function 'v'
        ...plugged/telescope.nvim/lua/telescope/builtin/__files.lua:529: in function 'v'
        ...im/plugged/telescope.nvim/lua/telescope/builtin/init.lua:514: in function <...im/plugged/telescope.nvim/lua/telescope/builtin/init.lua:483>
        ...ah/.vim/plugged/telescope.nvim/lua/telescope/command.lua:188: in function 'run_command'
        ...ah/.vim/plugged/telescope.nvim/lua/telescope/command.lua:253: in function 'load_command'
        ...e/sabah/.vim/plugged/telescope.nvim/plugin/telescope.lua:109: in function <...e/sabah/.vim/plugged/telescope.nvim/plugin/telescope.lua:108>
Press ENTER or type command to continue

### Neovim version

```markdown
NVIM v0.9.0-dev-454+g6d7b94ea0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc-10 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/runner/work/neovim/neovim/build/cmake.config -I/home/runner/work/neovim/neovim/src -I/home/runner/work/neovim/neovim/.deps/usr/include -I/usr/include -I/home/runner/work/neovim/neovim/build/src/nvim/auto -I/home/runner/work/neovim/neovim/build/include
Compiled by runner@fv-az435-184

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/share/nvim"

Run :checkhealth for more info
```

### Operating system and version

Ubuntu 22.04.1 LTS

### Telescope version / branch / rev

Telescope 0.1.0

### checkhealth telescope

```markdown
==============================================================================
telescope: require("telescope.health").check()

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 13.0.0
- OK fd: found fd 8.3.1

===== Installed extensions ===== ~
```

### Steps to reproduce

1. `nvim -nu minimal.lua`  (I'm using [vim-plug](https://github.com/junegunn/vim-plug) please install it. 
2.Try to open a lua (or any file where highlight is enabled by treesitter. In this case I have enabled it for lua in the minimal config.) buffer and run `:Telescope current_buffer_fuzzy_find` 

### Expected behavior

Live fuzzy search inside of the currently open buffer should work.

### Actual behavior

`Error executing Lua callback: ...plugged/telescope.nvim/lua/telescope/builtin/__files.lua:413: attempt to call method '_get_hl_from_capture' (a nil value)
stack traceback:
        ...plugged/telescope.nvim/lua/telescope/builtin/__files.lua:413: in function 'v'
        ...plugged/telescope.nvim/lua/telescope/builtin/__files.lua:529: in function 'v'
        ...im/plugged/telescope.nvim/lua/telescope/builtin/init.lua:514: in function <...im/plugged/telescope.nvim/lua/telescope/builtin/init.lua:483>
        ...ah/.vim/plugged/telescope.nvim/lua/telescope/command.lua:188: in function 'run_command'
        ...ah/.vim/plugged/telescope.nvim/lua/telescope/command.lua:253: in function 'load_command'
        ...e/sabah/.vim/plugged/telescope.nvim/plugin/telescope.lua:109: in function <...e/sabah/.vim/plugged/telescope.nvim/plugin/telescope.lua:108>`

### Minimal config

```Lua
local Plug = vim.fn["plug#"]

vim.call("plug#begin", "~/.vim/plugged")
Plug("nvim-lua/plenary.nvim")
Plug("nvim-telescope/telescope.nvim", { tag = '0.1.0' })
Plug("nvim-treesitter/nvim-treesitter", { ["do"] = ":TSUpdate" })
vim.call("plug#end")


require 'nvim-treesitter.configs'.setup {
    -- TS parsers
    ensure_installed = { "lua" },

    -- TS modules
    highlight = {
        enable = true,
    },

}
require('telescope').setup({
})

-- keymap("", "<Space>", "<Nop>", opts)
-- vim.g.mapleader = " "
-- vim.g.maplocalleader = " "
-- keymap("n", "<leader>fg", ":Telescope current_buffer_fuzzy_find<CR>", opts)
```

**Comment by Conni2461** [comment]
we somewhat addressed this in the current master version and the fix was backported to 0.1.x branch.

I should do a new release :thinking: 

Does it fix your issue, if you switch to that branch? If yes i'll close this issue.

There is this problem left https://github.com/nvim-telescope/telescope.nvim/issues/2192 where the highlighting isn't correctly done

**Comment by sabah1994** [comment]
Master branch did fix the issue and please do create a new release and update the documentation.

**Comment by dheimgartner** [comment]
I had the same problem: 0.1.x branch did not fix the issue but the master did...

So I guess the fix has not been backported?

**Comment by Conni2461** [comment]
hmm it is definitely backported: https://github.com/nvim-telescope/telescope.nvim/commit/51faca4b5303c9b679826050ef366869f458e670

**Comment by pprotas** [comment]
Hello, this is still broken:

```
E5108: Error executing lua: ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:413: attempt to call method '_get_hl_from_capture' (a nil value)
stack traceback:
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:413: in function 'v'
        ...im/lazy/telescope.nvim/lua/telescope/builtin/__files.lua:529: in function 'v'
        .../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:514: in function <.../nvim/lazy/telescope.nvim/lua/telescope/builtin/init.lua:483>
```

Using this config:
```lua
vim.keymap.set("n", "<leader>/", function()
	require("telescope.builtin").current_buffer_fuzzy_find(require("telescope.themes").get_dropdown({
		winblend = 10,
		previewer = false,
	}))
end, { desc = "[/] Fuzzily search in current buffer]" })
```

Telescope installed using Lazy:
```lua
{
	"nvim-telescope/telescope.nvim",
	version = "0.1.0",
	dependencies = "nvim-lua/plenary.nvim",
},
{
	"nvim-telescope/telescope-fzf-native.nvim",
	build = "make",
},

```

**Comment by Conni2461** [comment]
`version = "0.1.0"` is a tag, we can't backport fixes to a tag!

you want `0.1.x` and i gonna make a 0.1.1 version this weekend

**Comment by devstefancho** [comment]
I also had same problem, and fixed after update to 0.1.1
```lua
-- Telescope
use({ "nvim-telescope/telescope.nvim", tag = "0.1.1", requires = { { "nvim-lua/plenary.nvim" } } })
```
